tests/testthat/test-splatPop-simulate.R

```diff
@@ -0,0 +1,35 @@
+context("splatPopSimulate")
+
+library(vcfR)
```

Comment:
I don't think we need this library call

Reply:
Without loading the package I get the following error during testing:

```
test-splatPop-simulate.R:8: warning: (unknown)
data set 'vcfR_example' not found

test-splatPop-simulate.R:8: error: (unknown)
object 'vcf' not found
```

Reply:
This might be from the `data(vcfR_example)` call, you probably need `data(vcfR_example, package = "vcfR")`.

Reply:
This is still required for the tests to run (although now I load VariantAnnotation)